test_repo/test_module/website_templates.xml

```diff
@@ -7,11 +7,10 @@
             <span t-esc="price" t-esc-options='{"widget": "monetary"}'/>
         </div>
         <section class="oe_structure">
-            <p>Customize this!</p>
+            <p>Customize this! Not invalid because there is content here!</p>
```

Comment:
I'm not sure about this change. If the oe_structrure has some default content, isn't it still safer to spawn a child view to modify it?

Reply:
The reasoning was that there may exist XPaths that depend on the content, so making edits moves everything into a new child view, which may break the previous XPaths.

I don't know how often this happens/needs to happen, since I am coming to the conclusion that you can't really count on anything inside a section with modifiable content, so having XPaths that depend on it seems dangerous (users can delete the whole thing and break XPaths as well).

Reply:
Indeed, anything that depends on the contents of an editable section is broken by design. I don't think you need to handle this case.

BTW, according to odoo inheritance rules, if you're doing that, it's gonna break anyway when the user edits that part.